### DISCUSSION TITLE
Allow setting different colors

### DISCUSSION THREAD

**Discussion by M7amma** [jocmp/capyreader#403] [Feature Requests]
Please make it possible to allow changing colors for different elements like the feed item's title and etc

**Comment by montechristos** [comment]
I agree. I find the current colors quite hard on the eyes.
(as a reference, before capy readers, I have been using flym for many years. The colors there are easier on the eyes - my opinion)

**Comment by jocmp** [comment]
I don't want to forget about this - what version of Android are you using? In Android 12 and later I've deferred color styling to [Material You](https://source.android.com/docs/core/display/material), though I don't know if this varies by manufacturer.

**Reply by montechristos** [reply]
On my phone, I use Android 11 (I know, I need to update - LineageOS)
This is where I find the colors difficult on the eyes.
![CapyScreenshot](https://github.com/user-attachments/assets/1f031e00-f10c-4575-9a7b-13951ed3a259)
